src/jobs/helpers/jobVerifier.js

```diff
@@ -68,3 +75,26 @@ module.exports.verifyTestExists = async (req, res, next) => {
     }
     next(errorToThrow);
 };
+
+module.exports.verifyExperimentsExist = async (req, res, next) => {
+    const jobBody = req.body;
+    let errorToThrow;
+    const jobPlatform = await configHandler.getConfigValue(CONFIG.JOB_PLATFORM);
+    const experiments = jobBody.experiments;
+    if (!experiments || experiments.length === 0) {
+        next();
+    } else if (experiments.length > 0 && jobPlatform.toUpperCase() !== KUBERNETES) {
+        errorToThrow = new Error(ERROR_MESSAGES.CHAOS_EXPERIMENT_SUPPORTED_ONLY_IN_KUBERNETES);
+        errorToThrow.statusCode = 400;
+        next(errorToThrow);
+    } else {
+        const uniqueExperimentIds = [...new Set(experiments.map(experiment => experiment.experiment_id))];
```

Comment:
on the job there can be two different expiremts with the same id, but with different start time and end time?

```suggestion
        const uniqueJobExperimentIds = [...new Set(experiments.map(experiment => experiment.experiment_id))];
```

Reply:
yes :) For example if you have job of 3 hours, you can fail same pod for 15 mins at the begining, then stop and then do it again after 1 hour. It's same experiment (template)